fintech-examples/fintech-ui/src/app/bank/bank-routing.module.ts

```diff
@@ -10,6 +10,10 @@ const routes: Routes = [
       {
         path: 'account',
         loadChildren: () => import('./list-accounts/list-accounts.module').then(m => m.ListAccountsModule)
+      },
+      {
+        path: 'payment',
+        loadChildren: () => import('../payment/payment.module').then(m => m.PaymentModule)
```

Comment:
why is payment not below bank, like path would suggest? - this is just a question, not an advice!

Reply:
It was a mistake. fixed